.ci/azure-pipelines/build/ubuntu.yaml

```diff
@@ -1,4 +1,6 @@
 steps:
+  - script: (/usr/bin/Xvfb :99 -screen 0 1280x1024x24 -nolisten tcp -nolisten unix &)
```

Comment:
No need of using `/usr/bin` since it should be found by `which Xvfb` by default.

Let's also add a `nohup` before the command, just to make sure it keeps running